sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/aad/AADAutoConfiguration.java

```diff
@@ -20,15 +21,16 @@
  */
 @Configuration
 @ConditionalOnProperty(value = "spring.cloud.azure.active-directory.enabled", havingValue = "true")
-@EnableConfigurationProperties({
-    AADAuthenticationProperties.class,
-    AADResourceServerProperties.class
-})
 @Import({
+    AzureGlobalPropertiesAutoConfiguration.class,
```

Comment:
Why import this configuration here?

Reply:
There will be `NoSuchBeanDefinitionException` error if we delete this.

![image](https://user-images.githubusercontent.com/13167207/143995979-8e2a5eae-1e3a-4cef-a7bc-a6449e93ddb6.png)


![image](https://user-images.githubusercontent.com/13167207/143996054-4bca2190-ddcb-4b10-a88d-07a0bb3072fb.png)

Reply:
use constructor injection in the aad autoconfiguration class, and provide a bean in your test.